grafana-plugin/src/pages/incidents/Incidents.tsx

```diff
@@ -304,6 +304,8 @@ class _IncidentsPage extends React.Component<IncidentsPageProps, IncidentsPageSt
 
   renderIncidentFilters() {
     const { query, store } = this.props;
+    const defaultStart = moment().subtract(7, 'days');
+    const defaultEnd = moment().add(1, 'days');
```

Comment:
Using this instead of the 'now-7' options since the +1 day option is not supported.  The +1 day is to make sure we keep loading new alert groups that arrive after the table has loaded